radar/techniques.js

```diff
@@ -181,65 +167,71 @@ const content = [
         <li><i>unknown</i></li>
       </ul>
       <br/>
+      <br/>
       <h4>Cons:</h4>
       <ul>
         <li><i>unknown</i></li>
       </ul>
       <br/>
+      <br/>
       <h4>Conclusion</h4>
       <p>
-        Property-based testing is a robust and versatile testing methodology that complements traditional example-based testing.<br/>
+        Property-based testing is a robust and versatile testing methodology that complements traditional example-based testing.
         We would like to trial this technique and give a better judgment in a follow-up techradar.
       </p>
-    `
+    `,
   },
   {
-    name: "Mutation testing",
-    ring: "Hold",
-    quadrant: "Techniques",
-    isNew: "FALSE",
-    status: "No Change",
+    name: 'Mutation testing',
+    ring: 'Hold',
+    quadrant: 'Techniques',
+    isNew: 'FALSE',
+    status: 'No Change',
     description: `
       <h4>Description</h4>
       <p>
-        Mutation testing is conceptually quite simple.<br/>
-        Faults (or mutations) are automatically seeded into your code, then your tests are run. If your tests fail then the mutation is killed, if your tests pass then the mutation lived.<br/>
-        The quality of your tests can be gauged from the percentage of mutations killed.
+        Conceptually, mutation testing is straightforward. Using specialized tools, small changes (mutations) are made to the source code of an application, creating a "mutant." When tests are executed against this mutant, the expectation is that one or more tests will fail, indicating that the mutation introduced an incorrect system behavior. This process is referred to as "killing the mutant." If no tests fail, it is a clear indication that the mutated part of the application lacks sufficient test coverage. Mutation testing is often referred to as "testing the tests."
       </p>
       <br/>
       <p>
-        A short list of mutation testing tools (incomplete)
+        Here are a few examples of mutation testing tools (not an exhaustive list):
         <ul>
-          <li>PIT (pitest.org)</li>
-          <li>Arcmutate (arcmutate.com)</li>
-          <li>Stryker Mutator (stryker-mutator.io)</li>
+          <li>• PIT (pitest.org)</li>
+          <li>• Arcmutate (arcmutate.com)</li>
+          <li>• Stryker Mutator (stryker-mutator.io)</li>
```

Comment:
```suggestion
          <li>PIT (pitest.org)</li>
          <li>Arcmutate (arcmutate.com)</li>
          <li>Stryker Mutator (stryker-mutator.io)</li>
```

Graag ook echte links maken van de urls